### PR TITLE
Fixed typos

### PR DESCRIPTION
**Pull request by fritzrehde**
I fixed some typos that I found with [`typos`](https://github.com/crate-ci/typos).
I'm not sure about all of my fixes, so I have highlighted those places in some following messages.

### CODE REVIEW

File: include/modules/sway/language.hpp [hunk 1]
```diff
@@ -21,7 +21,7 @@ class Language : public ALabel, public sigc::trackable {
   auto update() -> void override;
 
  private:
-  enum class DispayedShortFlag { None = 0, ShortName = 1, ShortDescription = 1 << 1 };
+  enum class DisplayedShortFlag { None = 0, ShortName = 1, ShortDescription = 1 << 1 };
```
Comment (fritzrehde) [inline include/modules/sway/language.hpp:24]: You mean `Displayed` here, right?

File: protocol/dwl-ipc-unstable-v2.xml [hunk 2]
```diff
@@ -36,8 +36,8 @@ I would probably just submit raphi's patchset but I don't think that would be po
     </request>
 
     <request name="get_output">
-      <description summary="get a dwl_ipc_outout for a wl_output">
```
Comment (fritzrehde) [inline protocol/dwl-ipc-unstable-v2.xml:39]: Is `outout` some technical term from the protocol, or do you mean `output`?

Comment (Alexays) [reply protocol/dwl-ipc-unstable-v2.xml:39]: I'll prefer to don't touch this file, as it's a protocol file.

File: include/modules/sway/language.hpp (hunk 1 above)
Comment (Alexays) [reply include/modules/sway/language.hpp:24]: Yep.

**Pull request closed without merging**